config/mock.go

```diff
@@ -423,11 +429,11 @@ func (m *MockConfig) GetIsDryRun() bool {
 	return m.DryRun
 }
 
-func (m *MockConfig) GetAddHostMetadataToTrace() bool {
+func (m *MockConfig) GetAddHostMetadataToTrace() (addHostMetadataToTrace bool, err error) {
 	m.Mux.RLock()
 	defer m.Mux.RUnlock()
 
-	return m.AddHostMetadataToTrace
+	return m.AddHostMetadataToTraceVal, m.AddHostMetadataToTraceErr
```

Comment:
If we are changing `return m.AddHostMetadataToTrace` to use `m.AddHostMetadataToTraceVal` as the return value, we will need to assign `AddHostMetadataToTraceVal` to true in `newStartedApp` in `app_test.go`